container/legacy_load.bzl

```diff
@@ -11,57 +11,56 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Rule for loading an image from 'docker save' tarball or the current
-   container_pull tarball format into OCI intermediate layout.
+"""Rule for importing an image from 'docker save' tarballs.
 
-This extracts the tarball amd creates a filegroup of the untarred objects in OCI layout.
+This extracts the tarball, examines the layers and creates a
+container_import target for use with container_image.
 """
 
+load(
+    "//container:legacy_pull.bzl",
+    _python = "python",
+)
+
 def _impl(repository_ctx):
-    """Core implementation of new_container_load."""
+    """Core implementation of legacy_container_load."""
```

Comment:
add docs at top level of this file (and others for py container_pull) indicating this is deprecated and should not be modified unless there are breaking changes that need to be integrated

Reply:
done